rlog.go

```diff
@@ -385,6 +385,17 @@ func Infof(format string, a ...interface{}) {
 	basicLog(levelInfo, notATrace, format, "", a...)
 }
 
+// Println prints a message if RLOG_LEVEL is set to INFO or lower.
```

Comment:
Could we have a comment in here then that this is mostly for compatibility, but that it isn't really encouraged? Also, why not set it to DEBUG? That won't break anything and it'll make people think about the levels they want if they really want those messages.